src/Jackett.Common/Definitions/acervos-cc-api.yml

```diff
@@ -0,0 +1,155 @@
+    - {id: 1, cat: Movies, desc: "Filmes"}
+    - {id: 2, cat: TV, desc: "TV Séries"}
+    - {id: 3, cat: Other, desc: "Cursos"}
+    - {id: 4, cat: Console, desc: "Jogos"}
+    - {id: 5, cat: PC, desc: "Aplicativos"}
+    - {id: 6, cat: TV/Anime, desc: "Animes"}
+    - {id: 7, cat: Other, desc: "Materiais de Apoio"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid, tvdbid, tmdbid]
+    movie-search: [q, imdbid, tmdbid]
+
+settings:
+  - name: apikey
+    type: text
+    label: APIKey
+  - name: info_key
+    type: info
+    label: About your API key
+    default: "Find or Generate a new API Token by accessing your <a href=\"https://acervos.cc/\" target =_blank>Acervos</a> account <i>My Security</i> page and clicking on the <b>API Token</b> tab."
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: created_at
+    options:
+      created_at: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: /api/torrents
+  method: get
+  inputs:
+    api_token: "{{ .Config.apikey }}"
+  error:
+    - selector: a[href*="/login"]
+      message:
+        text: "The API key was not accepted by {{ .Config.sitelink }}."
+
+search:
+  paths:
+    # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
+    # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
+    - path: "/api/torrents/filter"
+      response:
+        type: json
+
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ .Keywords }}"
+    $raw: "{{ if .Query.Season }}&seasonNumber={{ .Query.Season }}{{ else }}{{ end }}{{ if .Query.Ep }}&episodeNumber={{ .Query.Ep }}{{ else }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    sortField: "{{ .Config.sort }}"
+    sortDirection: "{{ .Config.type }}"
+    perPage: 100
+    page: 1
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\.", " "]
+    - name: re_replace
+      args: ["(?i)\\bS(\\d+)", "T$1"]
+
+  rows:
+    selector: data
+    attribute: attributes
+    count:
+      selector: meta.total
+
+  fields:
+    category:
+      selector: category_id
+    title:
+      selector: name
+      filters:
+        - name: re_replace
+          args:
+            - "\\bDUAL\\b"
+            - "pt-BR/en-US"
```

Comment:
The args syntax is incorrect. refer to https://github.com/Jackett/Jackett/wiki/Definition-format#re_replace-1
It's uncommon for Sonarr/Radarr and other torznab apps to use the language id as a tag
```suggestion
          args: ["(?i)\\bDUAL", "Portuguese English"]
```

Reply:
Hey, I tested that way, and it works well, instead of using the compact syntax I used the full one.
Also, the border with `\bDUAL\b` is to avoid matching something like DUAL[a-z] or something like that.
And another thing, shouldn't the language be specialized for the country, like Portuguese Brazilian instead of Portuguese?
You can see it working with the replacement here:
<img width="1446" alt="image" src="https://user-images.githubusercontent.com/51510/175830877-9f4eff28-1241-4188-b566-59f75899add2.png">

Reply:
>Hey, I tested that way, and it works well, instead of using the compact syntax I used the full one.

Ha!
Well, I learn something new. I had no idea our cardigann could handle that format.
OK, so if it works and you want to keep it then that's ok with me. I'll merge. Thanks.

Reply:
but about the pt-BR/en-US, I could not find any docs about it on how they recognize the language, do you know of something like that?

Reply:
I'm not a *Arr user, but from the way all the other indexers have been configured, the full language name is the more common, eg. Spanish, English, etc.
This from my default Sonarr install
![image](https://user-images.githubusercontent.com/6933240/175832876-92baef39-737b-4464-a8f7-d9e1ea973ab6.png)